linux_os/guide/services/nfs_and_rpc/nfs_configuring_clients/disabling_nfsd/service_nfs_disabled/rule.yml

```diff
@@ -39,5 +39,6 @@ platform: machine
 template:
     name: service_disabled
     vars:
-        servicename: nfs
+        servicename: nfs-server
```

Comment:
The macro service name in the macros in the `description` and `ocil` should also be updated with this change too, and wrapped in Jinja conditionals, as the service name differs for SLE15.

Reply:
Ah, that's a good point. Updated